extensions/interactions/NumericExpressionInput/directives/numeric-expression-input-interaction.component.html

```diff
@@ -3,3 +3,31 @@
 <div ng-show="!$ctrl.isCurrentAnswerValid()" class="alert alert-danger mt-2">
   <[$ctrl.warningText]>
 </div>
+<!--
+  Guppy enforces the 'white-space', 'height' and 'width' styles causing the non-responsive behaviour.
+  The changes listed below present a plausible solution for the problem, even though it may not
+  agree with the principles of design for this project. !important overrides the properties of 'white-space','height'
+  and 'width' which are given by, thus making the text contained within the box.
```

Comment:
This sentence seems incomplete.

Reply:
Fixed. I don't know what was going on in my mind when I made these changes.